nemo/collections/asr/modules/conv_asr.py

```diff
@@ -356,9 +356,21 @@ def output_types(self):
         )
 
     def __init__(
-        self, feat_in, num_classes, emb_sizes=[1024, 1024], pool_mode='xvector', init_mode="xavier_uniform",
+        self,
+        feat_in,
+        num_classes,
+        emb_sizes=[1024, 1024],
```

Comment:
Dont directly use lists here, use None and check for None below and create [1024, 1024] if None. Refer https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Done